package.json

```diff
@@ -28,11 +28,11 @@
     "posttest": "npm run lint"
   },
   "dependencies": {
+    "lodash": "^4.17.21",
```

Comment:
use `"lodash.isequalwith": "^4.4.0"` is enough, so you don't need to change `ConnectedRouter.js`  file

Reply:
Check this issue -> https://github.com/supasate/connected-react-router/pull/408 
It was caused by adding it to devDependecy, and if you remove it from devDependency, a test error occurs.
This is why I changed ```lodash.isequalwith``` to ```lodash```.

Reply:
> use `"lodash.isequalwith": "^4.4.0"` is enough, so you don't need to change `ConnectedRouter.js` file

I think your intention is to install lodash.isequalwith only one installation in the Dependency, not peerDependency and devDependency. write ??

Reply:
Yes, now the commit is OK.

Reply:
@Ivocin Thanks for the suggestion that helped us make better code.🙏